style/style.css

```diff
@@ -0,0 +1,302 @@
+/* red */
+/* gray */
+/* - #1885a6 */
+/* blue buttons */
+/* tan */
+/* blue links */
+/* gray */
+/* line 12, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+body {
+  color: #5a5a5a;
+  line-height: 1.6;
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 17px; }
+
+/* line 18, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+h1, h2, h3, h4, h5 {
+  color: #e03031;
+  font-family: Georgia, serif;
+  font-weight: normal;
+  line-height: 1em;
+  margin-top: 9px;
+  margin-bottom: 8px; }
+
+/* line 26, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+h1 {
+  font-size: 2.7em; }
+
+/* line 29, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+h2 {
+  font-size: 1.7em; }
+
+/* line 32, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+h3 {
+  color: #05a6cc;
+  font-family: Helvetica, Arial, sans-serif; }
+
+/* line 36, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+a {
+  color: #05a6cc;
+  text-decoration: none; }
+
+/* line 40, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+header, article, #faq, footer {
+  padding: 40px; }
+
+/* line 43, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+header {
+  position: relative; }
+  /* line 45, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  header > p {
+    font-size: 1.3em;
+    padding-bottom: 21px; }
+  /* line 49, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  header .teaser_content {
+    color: white;
+    position: absolute;
+    bottom: 232px;
```

Comment:
I probably should have used top. I switched to top in the media query because the image was so small that it was easier to go from the top, but there's no reason that both instances couldn't have been from the top.